### PR TITLE
DOCS-131 Valkey packages doc

### PR DESCRIPTION
**Pull request by nastena1606**
Created the doc site, added styles and scripts
Added install page
Updated index page

### CODE REVIEW

File: docs/index.md
Comment (martinrvisser) [inline docs/index.md:1]: Perhaps change "Not happy with Redis and wish to switch to Valkey?" to "Unhappy with the Redis license change and wish to switch to the open source alternative Valkey?"

File: docs/installation.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
+# Install Valkey from Percona repositories
+
+Valkey offers packages for a select number of Linux distributions. Specifically, you can install it from Fedora/EPEL yum repositories and as binary tarballs for both Ubuntu Bionic and Ubuntu Focal. Additionally, it is available as a Docker container.
```
Comment (martinrvisser) [inline docs/installation.md:3]: and several of RHEL derivatives: [AlmaLinux](https://almalinux.org/), [Rocky Linux](https://rockylinux.org/)

File: docs/migration.md
Comment (martinrvisser) [inline docs/migration.md:1]: "they should also apply for on-premises deployments" to " the principles are the same for any deployment method"

Comment (martinrvisser) [inline docs/migration.md:1]: Another "migration method" deals specifically with the Redis used as a cache use case. Do we want to cover this or do we consider it too obvious to mention?

File: docs/index.md
Comment (nastena1606) [reply docs/index.md:1]: Love your suggestion, updated

File: docs/installation.md (hunk 1 above)
Comment (nastena1606) [reply docs/installation.md:3]: Does it relate to packages?

File: docs/migration.md
Comment (nastena1606) [reply docs/migration.md:1]: Updated

File: docs/installation.md (hunk 1 above)
Comment (nastena1606) [reply docs/installation.md:3]: Updated

File: docs/migration.md
Comment (nastena1606) [reply docs/migration.md:1]: We've decided to omit it as this is app migration rather than db migration

**Pull request merged**